Dockerfile

```diff
@@ -7,11 +7,12 @@ RUN \
   apk add --no-cache \
     dumb-init \
     git && \
-  go get github.com/gohugoio/hugo && \
+  go get github.com/kardianos/govendor && \
+  govendor get github.com/gohugoio/hugo && \
   cd $GOPATH/src/github.com/gohugoio/hugo && \
   go install && \
   cd $GOPATH && \
-  rm -rf pkg src && \
+  rm -rf pkg src bin/govendor && \
```

Comment:
Hi @ellerbrock,

Thank you for your quick response.

The `$GOPATH/.cache` directory is created by `govendor`, though I can see that, unlike this pull request, you are not using actually `govendor` at https://hub.docker.com/r/ellerbrock/alpine-hugo/~/dockerfile/, hence no `$GOPATH/.cache`.  :wink:  (which personally I won't judge because, putting my Debian Developer hat on, both Debian and Fedora's Go packaging policy demands that we un-vendor everything, so govendor isn't used in packaging Hugo, but anyhow...)

I am now attempting to run `sudo docker build` to actually test your Dockerfile.  (My hard disk is quite full; hope I won't run out of space doing so, haha.)